packages/react-core/src/core/utils/credentials.ts

```diff
@@ -0,0 +1,53 @@
+type PasswordCredentials = {
+  name: string;
+  id: string;
```

Comment:
what are name and id used for?

Reply:
It's the credentialStorage API type. Check https://web.dev/security-credential-management-retrieve-credentials/#auto-sign-in for more details